src/prommis/solvent_extraction/tests/test_sx_model_dynamic.py

```diff
@@ -0,0 +1,262 @@
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Y"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["La"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Ce"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Pr"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Nd"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Sm"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Gd"].fix(1e-9)
+        m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Dy"].fix(1e-9)
+
+        m.fs.solex.mscontactor.aqueous_inlet_state[:].flow_vol.fix(4.4)
+
+        m.fs.solex.mscontactor.aqueous[0, :].flow_vol.fix(4.4)
+
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Al"].fix(7.54e-10)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ca"].fix(4.955e-9)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Fe"].fix(1.491e-7)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sc"].fix(321.34)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Y"].fix(5.67e-6)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["La"].fix(1.78e-05)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ce"].fix(4.019e-5)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Pr"].fix(6.73e-6)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Nd"].fix(1.82e-5)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sm"].fix(3.285e-6)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Gd"].fix(1.55e-6)
+        m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Dy"].fix(9e-7)
+
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Al"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ca"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Fe"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sc"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Y"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["La"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ce"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Pr"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Nd"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sm"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Gd"].fix(1e-9)
+        m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Dy"].fix(1e-9)
+
+        m.fs.solex.mscontactor.organic_inlet_state[:].flow_vol.fix(62.01)
+
+        m.fs.solex.mscontactor.organic[0, :].flow_vol.fix(62.01)
+
+        return m
+
+    @pytest.mark.component
```

Comment:
AS this is a dynamic model, I think all of these should be `integration` tests rather than `component` tests. Integration is a tag we use to identify tests which will take a long time to run (such as dynamic simulations).

Reply:
This is only for the dynamic right? This is not for the steady state?

Reply:
Okay